lib/src/main/java/io/ably/lib/types/BaseMessage.java

```diff
@@ -131,11 +131,7 @@ public void encode(ChannelOptions opts) throws AblyException {
 					encoding = ((encoding == null) ? "" : encoding + "/") + "utf-8";
 				}
 			} else if(!(data instanceof byte[])) {
-				if (opts != null && opts.encrypted) {
 					throw AblyException.fromErrorInfo(new ErrorInfo("Invalid message data or encoding", 400, 40013));
```

Comment:
Surely we also need a whitespace change here as well?

Reply:
Yup. Fixed.